docs/src/main/sphinx/security/jwt-auth.rst

```diff
@@ -0,0 +1,145 @@
+==================
+JWT authentication
+==================
+
+Trino can be configured to authenticate client access using `JSON web tokens
+<https://en.wikipedia.org/wiki/JSON_Web_Token>`_. A JWT is a small, web-safe
+JSON representation that contains the same cryptographic information as a
+certificate, including:
+
+-  Subject
+-  Valid time period
+-  Signature
+
+A `JWT <jwt.io>`_ is designed to be passed between servers as proof of prior
+authentication. An end-user logs into a client application and requests access
+to a server. The server sends the user's credentials to a separate
+authentication service that validates the user, generates a JWT as proof of
+validation, then returns the JWT to the requesting server. The same JWT can then
+be forwarded to other services to maintain the user's validation without further
+credentials.
+
+.. important::
+
+    Do not configure JWT authentication in the belief that it is a part of
+    OAuth2 authentication.
+
+    Trino provides JWT authentication to support administrators at large sites
+    who are writing their own single sign-on or proxy system to stand between
+    users and the Trino coordinator, where their new system submits queries on
+    behalf of users.
+
+    JWT authentication is **not** expected to be needed or used by individuals
+    trying to connect to a Trino server.
+
+    Further, if another Trino-supported authentication system uses JSON web
+    tokens internally, such as :doc:`OAuth2 </security/oauth2>`, customers do
+    **not** need to configure JWT authentication as described on this page in
+    order to use that other authentication system. Use only the configuration
+    described on the other system's pages.
+
+Using JWT authentication
+------------------------
+
+A JSON web token can be passed between servers either in text form or encrypted
```

Comment:
I'd use JWT, it has already been spelled out.

Reply:
Grumble. That was me wanting to emphasize that it's a TOKEN but not wanting to be redundant with "JWT token". I'll drop back to JWT and hope the reader gets it. PS. "JWT" has a pronunciation: "jot". Who knew?